workflows/dragen-wts-qc-pipeline/3.9.3/dragen-wts-qc-pipeline__3.9.3.cwl

```diff
@@ -0,0 +1,204 @@
+cwlVersion: v1.1
+class: Workflow
+
+# Extensions
+$namespaces:
+    s: https://schema.org/
+$schemas:
+  - https://schema.org/version/latest/schemaorg-current-http.rdf
+
+# Metadata
+s:author:
+    class: s:Person
+    s:name: Sehrish Kanwal
+    s:email: sehrish.kanwal@umccr.org
+
+# ID/Docs
+id: dragen-wts-qc-pipeline--3.9.3
+label: dragen-wts-qc-pipeline v(3.9.3)
+doc: |
+  Workflow takes in dragen param along with object store version of a fastq_list.csv equivalent.
+  See the fastq_list_row schema definitions for more information.
+  Additonally runs qualimap step to genrate QC metrics.
```

Comment:
So this workflow calls the qualimap step

But the dragen-transcriptome-pipeline (subworkflow) also now calls the qualimap step? 

So the qualimap step is now called twice?

Reply:
Can you point where in the code you have noted a sub workflow call?

Reply:
Ah so the dragen wts qc pipeline only calls the dragen wts tool? 

So what's the difference between the qc pipeline and the regular wts pipeline?

Reply:
> Ah so the dragen wts qc pipeline only calls the dragen wts tool?

And qualimap step.

> So what's the difference between the qc pipeline and the regular wts pipeline?

We are not running additional steps, which are part of WTS pipeline (arriba, somalier, multiQC) as these are not needed to generate QC level data for multiQC reference samples. This will also be useful going forward, if we need to regenerate this data with new Dragen version.